v2/openapi/fb-provider-crosstransfer-api.yaml

```diff
@@ -0,0 +1,747 @@
+        '400':
+          $ref: '#/x-responses/BadRequestResponse'
+        '401':
+          $ref: '#/x-responses/UnauthorizedResponse'
+        '4XX':
+          $ref: '#/x-responses/ErrorResponse'
+        '5XX':
+          $ref: '#/x-responses/ErrorResponse'
+
+
+  /accounts/{accountId}/crosstransfers/standingorders/offramp:
+    post:
+      operationId: createOffRampStandingOrder
+      summary: >-
+        Creates deposit-only crypto account as a source, configures trigger and stores 
+        payment instruction details about destination fiat account. As a result, the user will be
+        able to deposit crypto to the source account and receive fiat to the destination account.
+      tags: [ crosstransfers ]
+      parameters:
+        - $ref: '#/x-header-params/X-FBAPI-KEY'
+        - $ref: '#/x-header-params/X-FBAPI-NONCE'
+        - $ref: '#/x-header-params/X-FBAPI-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-TIMESTAMP'
+        - in: path
+          name: accountId
+          required: true
+          description: Sub-account identifier.
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateOffRampRoutingRequest'
+      responses:
+        '200':
+          description: Off ramp standing order with crypto source sub-account created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OffRampStandingOrder'
+        '400':
+          $ref: '#/x-responses/BadRequestResponse'
+        '401':
+          $ref: '#/x-responses/UnauthorizedResponse'
+        '4XX':
+          $ref: '#/x-responses/ErrorResponse'
+        '5XX':
+          $ref: '#/x-responses/ErrorResponse'
+
+  /accounts/{accountId}/crosstransfers/standingorders/offramp/{id}:
+    get:
+      operationId: getOffRampStandingOrder
+      summary: >-
+        Deletes the standing order.
+      tags: [ crosstransfers ]
+      parameters:
+        - $ref: '#/x-header-params/X-FBAPI-KEY'
+        - $ref: '#/x-header-params/X-FBAPI-NONCE'
+        - $ref: '#/x-header-params/X-FBAPI-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-TIMESTAMP'
+        - $ref: '#/x-path-params/SubAccountIdPathParam'
+        - in: path
+          name: id
+          required: true
+          description: Standing order identifier.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Standing order details.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OffRampStandingOrder'
+        '400':
+          $ref: '#/x-responses/BadRequestResponse'
+        '401':
+          $ref: '#/x-responses/UnauthorizedResponse'
+        '4XX':
+          $ref: '#/x-responses/ErrorResponse'
+        '5XX':
+          $ref: '#/x-responses/ErrorResponse'
+    put:
```

Comment:
Lets remove update action. User in future should be able to archive standing order and to create new one if needed.